### DISCUSSION TITLE
Should "ADB-Enhanced" project belong to Mobile Native Foundation?

### DISCUSSION THREAD

**Discussion by ashishb** [MobileNativeFoundation/discussions#14] [Proposals]
https://github.com/ashishb/adb-enhanced is a tool for testing and development while building Android apps. Should it be part of the Mobile Native Foundation?

**Comment by keith** [comment]
In general the way the Mobile Native Foundation will accept new projects is by gauging community interest, and then having the committee vote on it (there are more details on this [here](https://github.com/MobileNativeFoundation/foundation)). 

Since we're still quite new we haven't fully fleshed out this process (as the committee hasn't even had the first meeting yet). But I think having a thread like this is the right place to start, so I would be interested in hearing from some more android folks on the specifics of this project.

**Comment by ashishb** [comment]
@keith has the process been flushed out now? Any recommendations on how to proceed with making `ADB-Enhanced` a project of the Mobile Native Foundation?

**Reply by keith** [reply]
Ah sorry, there is a full doc on this process now https://github.com/MobileNativeFoundation/foundation/blob/main/project-lifecycle.md

**Reply by ashishb** [reply]
- [x] adb-enhanced is open-sourced under Apache License
- [x] The project does have a community interest as indicated by the number of downloads - https://pepy.tech/project/adb-enhanced
- [x] adb-enhanced is useful for every Android developer
- [x] adb-enhanced is production-ready including a comprehensive set of tests that run on 10 different Android versions
- [x] adb-enhanced has only one maintainer that's me (`ashishb`) for now

How should I submit the project for evaluation by TSC?

**Reply by ashishb** [reply]
@keith pinging again. Maybe I missed something but I am unable to see how to submit the project to Mobile Native Foundation.

**Reply by keith** [reply]
Hey @ashishb, @tyvsmith was actually looking into this project so hopefully he can reply here

**Comment by tyvsmith** [comment]
We've been doing some research into the project to see if it'd be a good fit. Would you be interested in taking a video call to discuss the project? If so, should we connect via email to get that going?

**Reply by ashishb** [reply]
Definitely @tyvsmith. My email is ashishb[AT]ashishb.net.

**Comment by tyvsmith** [comment]
Sharing here for community visibility. After meeting with @ashishb and aligning within the TSC we've decided against taking this project into the org for now, but will keep an eye on it's growth, standardization, and maturity over time to see if it's a good fit in the future.

In the meantime, we encourage continued participation and contribution to MNF projects and discussions!
